lms/static/js/spec/learner_dashboard/course_card_view_spec.js

```diff
@@ -108,8 +108,12 @@ define([
                 expect(view.$('.course-details .course-text .course-key').html()).toEqual(context.key);
                 expect(view.$('.course-details .course-text .run-period').length).toBe(0);
                 expect(view.$('.no-action-message').text().trim()).toBe('Coming Soon');
-                expect(view.$('.enroll-open-date').text().trim())
-                    .toBe(context.run_modes[0].enrollment_open_date);
+                expect(view.$('.enrollment-opens').text().trim())
+                    .toEqual('Enrollment Opens on ' + courseCardModel.replaceAll(
+                        context.run_modes[0].enrollment_open_date,
+                        ' ',
+                        String.fromCharCode(160))
```

Comment:
Can you use a variable `nbsp = 160` instead of using the char code directly? Much more readable that way.